.gitignore

```diff
@@ -240,6 +240,7 @@
 
 # VS Code
 .vscode
+.vs
```

Comment:
`.vs` (as well as `.vscode`) is a directory, isn't it? Then it would be better to write it as `.vs/`